charters/DataStore/charter.md

```diff
@@ -0,0 +1,75 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at PB scale, with strong public APIs. It provides a simple API for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
+The objective of the Data Store group is to deliver substantively complete functionality on all of the in-scope items listed in this charter.
+
+## In-scope
+
+### Interfaces
+* DSS data read and write API (PUT bundle, PUT file, GET bundle, GET file) - maintenance and extension of the implementation of the basic data access APIs.
+* Checkout service APIs - Provide continuing support for the ability to checkout the data to a local filesystem, or a personal cloud environment
+* Collections service APIs - Maintenance and extension of the ability to do basic operations on arbitrary collections of objects in the Data Store.
```

Comment:
Or:

* Maintain and extend the **Collections service API** which enables basic operations on arbitrary collections of objects in the Data Store.